lib/swaps/Swaps.ts

```diff
@@ -582,7 +624,11 @@ class Swaps extends EventEmitter {
       await peer.sendPacket(new packets.SwapCompletePacket(responseBody));
     } catch (err) {
       this.failDeal(deal, SwapFailureReason.SendPaymentFailure, err.message);
-      await this.sendErrorToPeer(peer, rHash, err.message);
+      await this.sendErrorToPeer({
+        peer,
+        rHash,
+        errorMessage: err.message,
```

Comment:
Similarly we can set `SwapFailureReason.SendPaymentFailure` for the reason here.